### PR TITLE
feat(select): update select component to include radix primitives for scrolling on large datasets.

### PR DESCRIPTION
**Pull request by iaingymware**
I have added an up and down chevron to the select content area for when too many select items exist to display within the viewport.  This is taking the radix ScrollUpButton and ScollDownButtons from https://www.radix-ui.com/primitives/docs/components/select

image

image

As can be seen, the chevrons will only appear when a large number of SelectItems exist.

I noticed updating the registry added like 80 files to the PR so I removed it (the 3rd and 4th commit).  I am therefore just updating two files, the default and newyork select components.

If the registry update is required, happy to receive direction on how best to do this.

Thanks

### CODE REVIEW

**Comment by iaingymware** [comment]
Merged latest main into PR branch as was outdated.

File: apps/www/registry/new-york/ui/select.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import * as React from "react"
 import { CaretSortIcon, CheckIcon } from "@radix-ui/react-icons"
 import * as SelectPrimitive from "@radix-ui/react-select"
+import { ChevronDown, ChevronUp } from "lucide-react"
```
Comment (shadcn) [inline apps/www/registry/new-york/ui/select.tsx:6]: For `new-york`, let's use icons from `@radix-ui/react-icons`

Comment (iaingymware) [reply apps/www/registry/new-york/ui/select.tsx]: Updated as per request & commited.

**Comment by iaingymware** [comment]
@shadcn  awaiting review.

**Comment by iaingymware** [comment]
Updated branch as outdated.

**Comment by iaingymware** [comment]
Is this PR now complete?  I don't see that it has been reviewed yet.  Is there any action pending on me?

**Pull request merged**